### PR TITLE
Add resize observer to ignored errors in Sentry

### LINKED ISSUE #2587: Add "ResizeObserver loop limit exceeded" to ignored errors in Sentry
## Description

We get a lot of these errors in Sentry. They seem to be harmless, and we've ignored them in Sentry dashboard. To prevent them from filling up our Sentry quota, we should add them to the ignored errors when initializing Sentry in Nuxt:
```
Sentry.init({
   ...
   ignoreErrors: ['ResizeObserver loop limit exceeded'],
 })
```
(from https://forum.sentry.io/t/resizeobserver-loop-limit-exceeded/8402/4)

## Sentry link

Sentry Issue: [OPENVERSE-FRONTEND-ZM](https://openverse.sentry.io/issues/4292069723/?referrer=github_integration)

```
Error: ResizeObserver loop limit exceeded
  at None (/search/)

ResizeObserver loop limit exceeded
```

### PR DESCRIPTION
**Pull request by heyligerjon**
## Fixes

Fixes #2587 by @obulat

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by krysal — APPROVED** [review]
Looks good. Thank you for your help @heyligerjon!

File: frontend/src/utils/sentry-config.ts [hunk 1]
```diff
@@ -21,6 +21,9 @@ export const sentryConfig: ModuleOptions = {
       // Ignore errant focus-trap-vue errors
       /`initialFocus` did not return a node/,
 
+      // Ignore ResizeObserver loop limit exceeded errors
+      /ResizeObserver loop limit exceeded/,
```
Comment (obulat) [inline frontend/src/utils/sentry-config.ts:25]:
```suggestion
      /ResizeObserver loop limit exceeded/,
      /ResizeObserver loop completed with undelivered notifications/,
```

File: frontend/src/utils/sentry-config.ts [hunk 2]
```diff
@@ -21,6 +21,9 @@ export const sentryConfig: ModuleOptions = {
       // Ignore errant focus-trap-vue errors
       /`initialFocus` did not return a node/,
 
+      // Ignore ResizeObserver loop limit exceeded errors
```
Comment (obulat) [inline frontend/src/utils/sentry-config.ts:24]:
```suggestion
      // Ignore ResizeObserver loop-related errors
```

**Review by obulat — APPROVED** [review]
Thank you for your contribution, @heyligerjon! 🎉  I've added another error that's related to ResizeObserver, and will merge the PR after the CI passes :)

**Pull request merged**